### DISCUSSION TITLE
integration whatapp

### DISCUSSION THREAD

**Discussion by jonasbrant** [exoplatform/exo-community#19] [Q&A]
We usualy recieve messagens in whatapp and because it is not easy to foward we tend to comunicate using whatapp
A possible integration and messages forward from whatapp to exo chat could be a great oportunity to stimulate exo chat use.

**Comment by rdenarie** [comment]
Thanks for you proposition.

In fact, we have plan to rework our chat module in order to make it more extensible with connectors. 
After this rework, an integration could be possible with Whatsapp
